pkg/server/api/entry.go

```diff
@@ -171,6 +167,16 @@ func ProtoToRegistrationEntryWithMask(ctx context.Context, td spiffeid.TrustDoma
 		storeSVID = e.StoreSvid
 	}
 
+	var x509SvidTTL int32
+	if mask.X509SvidTtl {
+		x509SvidTTL = e.X509SvidTtl
+	}
+
+	var jwtSvidTTL int32
+	if mask.JwtSvidTtl {
+		jwtSvidTTL = e.JwtSvidTtl
+	}
+
```

Comment:
x509SVIDTTL (jwtSVIDTTL) or in this case, since they are very locally used explicitly for setting fields in the proto, I'd also be ok if they matched the proto convention. Right now they are a hybrid :)

Reply:
According to the linting rules, these variables must be named in a particular way. Is it alright if I leave them as is so they pass lint?

```
pkg/server/api/entry.go:170:6: var-naming: var x509SvidTtl should be x509SvidTTL (revive)
	var x509SvidTtl int32
	    ^
pkg/server/api/entry.go:175:6: var-naming: var jwtSvidTtl should be jwtSvidTTL (revive)
	var jwtSvidTtl int32
	    ^
make: *** [lint-code] Error 1
```

Reply:
yeah, satisfy the linter.